### DISCUSSION TITLE
Reset CC2530 router without re-flashing?

### DISCUSSION THREAD

**Discussion by aptonline** [Koenkk/zigbee2mqtt#23810] [Devices]
I've made a router from the instructions [here](https://www.zigbee2mqtt.io/advanced/zigbee/05_create_a_cc2530_router.html) and it's working perfectly but I've recently changes my Zibbee network and need to move the router to the new network. 

Is there anyway to reset the network config on the CC2530 without having to refresh the board?  I've tried the reset pin (GND+Reset) but I think that just reboots the device.

**Comment by francisp2** [comment]
To reset it into pairing mode power-cycle it three times as follows:

    power on
    wait 2sec
    power off
    repeat above steps 2 further times
    power on and wait for it to join your network
